### ISSUE TITLE
Build script crash

### ISSUE THREAD

**Issue by grouxsome** [Shippable/support#3518]
Seeing build script crashes again. Could not reopen previous issue (https://github.com/Shippable/support/issues/3500). Please note this build completes both locally and when run manually on the build node image.

`ERROR:script_runner - script_runner:Command failed : ssh-agent bash -c 'ssh-add /tmp/ssh/00_sub;ssh-add /tmp/ssh/01_deploy;ssh-add /tmp/ssh/GithubSSHKey; cd /root && /root/3559504f-7ac2-44b7-a13e-5652a187101c.sh'
Exception Invalid or no script tags received
ERROR:script_runner - script_runner:Command failed : ssh-agent bash -c 'ssh-add /tmp/ssh/00_sub;ssh-add /tmp/ssh/01_deploy;ssh-add /tmp/ssh/GithubSSHKey; cd /root && /root/3559504f-7ac2-44b7-a13e-5652a187101c.sh'`

https://app.shippable.com/github/zenreach/WiFast-Base/runs/2204/1/console

**Comment by ambarish2012** [comment]
@grouxsome just want to confirm that the console link you sent is accurate, since that build failed last wednesday. Have you seen more recent failures that were similar? I am investigating that error regardless.

**Comment by ambarish2012** [comment]
I am also seeing some failures in your recent builds in the develop branch specifically, looking into those as well so assess if multiple builds are being impacted by some product issue.

**Comment by grouxsome** [comment]
Hello,

Anything on that branch will fail (with my change only). Latest failure is
https://app.shippable.com/github/zenreach/WiFast-Base/runs/2351/1/console

We are reverting the change though so it will start building soon. The
change which causes the crash is executing a command which is copying files
from one dir to another.

**Comment by ambarish2012** [comment]
@grouxsome I am seeing a common pattern in all your failed builds.

Copying '/root/src/github.com/zenreach/WiFast-Base/wifast/links/static/css/blueprint/plugins/rtl/screen.css'

They are all failing at some point after copying screen.css.

Copying '/root/src/github.com/zenreach/WiFast-Base/third-party/dist-modified/Django/django/contrib/admin/media/img/gis/move_vertex_on.png'
Copying '/root/src/github.com/zenreach/WiFast-Base/third-party/dist-modified/Django/django/contrib/admin/media/img/admin/icon_searchbox.png'
Copying '/root/src/github.com/zenreach/WiFast-Base/third-party/dist-modified/Django/django/contrib/admin/media/img/admin/selector_stacked-remove.gif'
Copying '/root/src/github.com/zenreach/WiFast-Base/third-party/dist-modified/Django/django/contrib/admin/media/img/admin/selector-add.gif'
Copying '/root/src/github.com/zenreach/WiFast-Base/third-party/dist-modified/Django/django/contrib/admin/media/img/admin/icon_alert.gif'
Copying '/root/src/github.com/zenreach/WiFast-Base/third-party

A couple of questions -

1) Please can you check you have enough disk space, one suspicion i have is that build is running out of free disk space.

2) In a green build, I do not see any of these files being copied. Any idea why the copying is bypassed?

**Comment by ambarish2012** [comment]
@grouxsome since you are reverting the copy command, I am closing the ticket. Please reopen the ticket if you run into any more failures.

**Comment by grouxsome** [comment]
This project is transitioning from a legacy build/deploy system to
Shippable/Docker. The copying of these files is now required for the
project to function properly. I would assume green builds are builds
without my latest change to aggregate these static assets into a common
folder.

We are only reverting on the ONE branch to unblock dev. We NEED this change
to go to production.

New ticket: https://github.com/Shippable/support/issues/3519 because I
cannot re-open once you close.

**Comment by ambarish2012** [comment]
@grouxsome i apologize for closing the ticket, i was under the impression that you are reverting the copy command for good, but I now understand better.
